app/views/user_mailer/shared/_in_person_ready_to_verify.html.erb

```diff
@@ -10,7 +10,7 @@
   </tr>
 </table>
   <h1>
-    <%= t('in_person_proofing.headings.barcode') %>
+    <%= @presenter.barcode_heading_text %>
```

Comment:
This will only display when the outage message is enabled.  Thoughts?

Reply:
Good clean up